pkgs/development/node-packages/node-packages.json

```diff
@@ -289,6 +289,45 @@
 , "textlint-rule-unexpanded-acronym"
 , "textlint-rule-write-good"
 , "thelounge"
+, "thelounge-plugin-closepms"
+, "thelounge-plugin-giphy"
+, "thelounge-plugin-shortcuts"
+, "thelounge-theme-abyss"
+, "thelounge-theme-amoled"
+, "thelounge-theme-amoled-sourcecodepro"
+, "thelounge-theme-bdefault"
+, "thelounge-theme-bmorning"
+, "thelounge-theme-chord"
+, "thelounge-theme-classic"
+, "thelounge-theme-common"
+, "thelounge-theme-crypto"
+, "thelounge-theme-discordapp"
+, "thelounge-theme-dracula"
+, "thelounge-theme-dracula-official"
+, "thelounge-theme-flat-blue"
+, "thelounge-theme-flat-dark"
+, "thelounge-theme-gruvbox"
+, "thelounge-theme-hexified"
+, "thelounge-theme-ion"
+, "thelounge-theme-light"
+, "thelounge-theme-midnight"
+, "thelounge-theme-mininapse"
+, "thelounge-theme-monokai-console"
+, "thelounge-theme-mortified"
+, "thelounge-theme-neuron-fork"
+, "thelounge-theme-new-morning"
+, "thelounge-theme-new-morning-compact"
+, "thelounge-theme-nologo"
+, "thelounge-theme-nord"
+, "thelounge-theme-onedark"
+, "thelounge-theme-purplenight"
+, "thelounge-theme-scoutlink"
+, "thelounge-theme-seraphimrp"
+, "thelounge-theme-solarized"
+, "thelounge-theme-solarized-fork-monospace"
+, "thelounge-theme-zenburn"
+, "thelounge-theme-zenburn-monospace"
+, "thelounge-theme-zenburn-sourcecodepro"
```

Comment:
It would be awesome if they would be discoverable under `thelounge.plugins`.

Reply:
Do you suggest that I populate that set dynamically, or statically?

Reply:
If you have an idea how to accomplish that dynamically that would be neat. I'd be fine with referencing those manually.

Reply:
Is it worth increasing Nixpkgs evaluation time, though? I'm not sure.

Reply:
I wouldn't mind.